### ISSUE TITLE
Implement weapon inaccuracy

### ISSUE THREAD

**Issue by ClobberXD** [ClobberXD/gunslinger#17]
Implement inaccuracy, and implement increased accuracy when aiming down sights. This feature would replace the lame 20% additional damage when aiming down sights.

**Comment by ClobberXD** [comment]
f4cdc15 implements spread. What's left to do is to somehow reduce spread if player is aiming down sights.

**Comment by ClobberXD** [comment]
Bullet drop should also be implementable if using progresssive raycasting (#27) or entity-based projectile handling (#25).

**Comment by afshaan4** [comment]
do you want to reduce spread when scoped or remove spread when scoped?

**Comment by ClobberXD** [comment]
Yeah, reducing spread when the player is scoping is what I'd like to see. But ideally, the amount of reduction in spread (i.e. increase in accuracy) should be customisable per-weapon (or per-addon when addon support is added later).

**Comment by ClobberXD** [comment]
Forgot to note, recoil has been added in 79991790bf3a3d9b28768955d6cd2c6a3d03785a

Only bullet drop remains. I'll implement this after #27.
